src/types.js

```diff
@@ -25,5 +26,5 @@ export type Placement =
   | 'top-center'
   | 'top-right';
 
-export type ToastType = Options & { appearance: AppearanceTypes, content: Node, id: Id };
+export type ToastType = { content: Node, id: Id } & Options;
```

Comment:
This fixes a flow issue I was having where it thought `id` was optional on internal state.